src/app/weather-data/monthly-station-data/monthly-station-table/monthly-station-table.component.html

```diff
@@ -81,6 +82,11 @@
     </tfoot>
 </table>
 
+<div>
```

Comment:
lets have this button and the pagnation on the same row. "justify-content-between"